### PR TITLE
SFRA: unused code

### PR DESCRIPTION
**Pull request by njparadis**
Remove functions that are not used anywhere in the cartidge code and do not override core SFCC functions

### CODE REVIEW

**Review by klaviyojad — APPROVED** [review]
lgtm as long as it doesnt break anything when installed in a system

File: cartridges/int_klaviyo_sfra/cartridge/scripts/utils/klaviyo/klaviyoDataLayer.js
Comment (remstone7) [inline cartridges/int_klaviyo_sfra/cartridge/scripts/utils/klaviyo/klaviyoDataLayer.js:85]: this does look like the core cartridge calls this, just checking to make sure that this doesn't need to be overriden?

Comment (njparadis) [reply cartridges/int_klaviyo_sfra/cartridge/scripts/utils/klaviyo/klaviyoDataLayer.js:85]: Both of the places that buildCartDataLayer are called are pointing to the function in the core klaviyoUtils file:
https://github.com/klaviyo/SFCC_Klaviyo/blob/67764a60e47a01d6591688405cef3e0bb93e169e/cartridges/int_klaviyo/cartridge/controllers/Klaviyo.js#L52
https://github.com/klaviyo/SFCC_Klaviyo/blob/67764a60e47a01d6591688405cef3e0bb93e169e/cartridges/int_klaviyo_sfra/cartridge/controllers/Klaviyo.js#L54

**Review by remstone7 — APPROVED** [review]
Approved, but can we please add a changelog as well!

**Comment by njparadis** [comment]
> Approved, but can we please add a changelog as well!

Will rebase once #13 is merged

**Pull request closed without merging**
